README.md

```diff
@@ -157,6 +157,13 @@ fn main() {
     window.show();
 
     event_loop.run_forever(|_event| {
+        match _event {
```

Comment:
Rename `_event` to `event`.
Adding an underscore in from a variable means it's not used.

Reply:
Ok.